lib/JSON/Validator.pm

```diff
@@ -753,8 +753,6 @@ sub _validate_format {
   return E $path, [format => $schema->{format}, $err];
 }
 
-sub _validate_type_any { }
```

Comment:
I don't think this change is relevant for this PR? This also goes for the change in jv-basic.t.

Reply:
sure, I can pull it out and move it to a later PR if you like. The only use of the 'any' type occurs within array validation, so I thought it was reasonable to include it here.

Reply:
and done.